src/app/zap-templates/common/ClusterTestGeneration.js

```diff
@@ -733,7 +733,7 @@ function octetStringEscapedForCLiteral(value)
 {
   // Escape control characters, things outside the ASCII range, and single
   // quotes (because that's our string terminator).
-  return value.replace(/\p{Control}|\P{ASCII}|"/gu, ch => {
+  return value.replace(/\P{Control}|\p{Control}/gu, ch => {
```

Comment:
Looking https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Unicode_Property_Escapes#syntax it does indeed that \p and \P are negation of each other, so this does effectively `value.replace(/./gu, ch => { ...` which seems not intended.

Reply:
Hmmm.... You are correct. I will update the code so that when there is a non ASCII character to convert to binary string literal and when it is all ascii, just keep the value.

Reply:
Ok. Updated

Reply:
Thank you.